delete-publication-from-scopus/build.gradle

```diff
@@ -0,0 +1,25 @@
+def problemVersion = '0.24.0'
```

Comment:
Is this line necessary for some arcane reason, or has it been copied from another config?

If it is the former, please annotate to reflect this